src/simpleitk_timeseries_motion_correction/create_animation.py

```
@@ -158,9 +158,20 @@ def process_frame(
     return combined
 
 
-def main(input_file, output_file, additional_input_files=None, scale=2.0, fps=10):
-    print(f"Loading {input_file}...")
-    img = sitk.ReadImage(input_file)
+def main(input_img, output_file, additional_input_imgs=None, labels=None, scale=2.0, fps=10):
+
+    # prepare list of axis labels
+    if not labels is None:
```

Ruff (0.14.14)

[error] Test for object identity should be `is not`

Convert to `is not`

(E714)